plugins/scaffolder/api-report.md

```diff
@@ -252,6 +259,7 @@ export class ScaffolderClient implements ScaffolderApi {
   constructor(options: {
     discoveryApi: DiscoveryApi;
     fetchApi: FetchApi;
+    identityApi: IdentityApi;
```

Comment:
Perhaps we could keep this completely non-breaking by making this `identityApi?: IdentityApi` and throw an error if `listTasks` is called with `'owned'` parameter?